nixos/modules/services/web-servers/apache-httpd/default.nix

```diff
@@ -22,7 +22,11 @@ let
 
   php = cfg.phpPackage.override { apacheHttpd = pkg; };
 
-  phpMajorVersion = lib.versions.major (lib.getVersion php);
+  # If php version is version 8 or above it's just named libphp.so,
+  # but for previous versions it's named: libphp7.so or libphp5.so
+  phpMajorVersion = let
+    majorVersion = lib.versions.major (lib.getVersion php);
+  in (if majorVersion == "8" then "" else majorVersion);
```

Comment:
Since this no longer represents the major version in all cases, maybe it should be renamed / restructured? I'm thinking it could be called `phpModuleName` and be prepended with `php`, so `php5`, `php7` or `php`, depending on the version. That would make it clearer what it's used for and make sure it's not used accidentally where the actual major version is expected.

Reply:
Good point! Fixed in 1216baa :slightly_smiling_face: